lambda_functions/va_profile/va_profile_opt_in_out_lambda.py

```diff
@@ -7,20 +7,20 @@
 import os
 import psycopg2
 import sys
-from http.client import ConnectionError, HTTPSConnection
+from http.client import HTTPSConnection
 from json import dumps
 from ssl import SSLContext, SSLError
 
 
 OPT_IN_OUT_QUERY = """SELECT va_profile_opt_in_out(%s, %s, %s, %s, %s);"""
-NOTIFY_ENVIRONMENT = os.getenv("notify_environment")
-NOTIFICATION_API_DB_URI = os.getenv("notification_api_db_uri")
+NOTIFY_ENVIRONMENT = os.getenv("NOTIFY_ENVIRONMENT")
 PEM = None
-VA_PROFILE_DOMAIN = os.getenv("va_profile_domain")
+SQLALCHEMY_DATABASE_URI = os.getenv("SQLALCHEMY_DATABASE_URI")
```

Comment:
I updated the lambda infra to match this name.  Having the lambda environment variable name by the same as the the API code base simplifies testing.